fakeredis.py

```diff
@@ -312,6 +312,27 @@ def renamenx(self, src, dst):
         else:
             return self.rename(src, dst)
 
+    def scan(self, cursor=0, match=None, count=None):
+        cursor = int(cursor)
+        if count is None:
+            count = 10
+        match_keys = [key for key in self._db \
+            if not key or not match or \
+            fnmatch.fnmatch(to_native(key), to_native(match))]
+        keys = match_keys[cursor:cursor+count]
+        if cursor + count >= len(match_keys):
+            return 0, keys
+        else:
+            return cursor + count, keys[cursor:cursor+count]
```

Comment:
I believe this should just be `return cursor + count, keys` because you've already sliced the list on line 322.